pagopa/io-app
# fix: [IA-107] Missing accessibility role link on search bank privacy policy

## Short description
Added accessibility role on search bank privacy policy, if screen reader is enabled I added the possibility to press the Body component, so the user knows there is a link inside it.

## How to test
1. With the screen reader activated go to wallet -> add -> payment method -> PagoBANCOMAT card or App and digital payment tools.
2. For the link "privacy policy" the whole block of text should be read and behave like a link.

**Before:**

https://user-images.githubusercontent.com/61834253/126490240-e0893711-79a5-4f33-add2-aed8e31f80f8.mp4

**After:**

https://user-images.githubusercontent.com/61834253/126490294-bd793427-fd9c-4c3d-b30a-136114a2c464.mp4
